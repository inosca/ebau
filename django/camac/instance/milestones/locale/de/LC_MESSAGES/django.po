# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-09-11 17:25+0200\n"
"PO-Revision-Date: 2024-08-29 13:33+0200\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Poedit 3.0.1\n"

#: camac/instance/milestones/serializers.py
msgid "regular process"
msgstr "Ordentliches Verfahren"

#: camac/instance/milestones/serializers.py
msgid "instance submitted"
msgstr "Dossier erfasst"

#: camac/instance/milestones/serializers.py
msgid "Paper submission"
msgstr "Eingang Post"

#: camac/instance/milestones/serializers.py
msgid "Additional demand"
msgstr "Nachforderung Gemeindebaubehörde"

#: camac/instance/milestones/serializers.py
msgid "Check completed"
msgstr "Vollständigkeitsprüfung abgeschlossen"

#: camac/instance/milestones/serializers.py
msgid "Instance complete"
msgstr "Dossier vollständig"

#: camac/instance/milestones/serializers.py
msgid "Review building commission"
msgstr "Prüfung durch Gemeinde"

#: camac/instance/milestones/serializers.py
msgid "Puclication date"
msgstr "Publikationsdatum"

#: camac/instance/milestones/serializers.py
msgid "Receipt of objections"
msgstr "Eingang von Einsprachen"

#: camac/instance/milestones/serializers.py
msgid "Forwarding to KOOR"
msgstr "Weiterleitung an KOOR"

#: camac/instance/milestones/serializers.py
msgid "Start circulation"
msgstr "Start Zirkulation"

#: camac/instance/milestones/serializers.py
msgid "Distribution completed"
msgstr "Zirkulation beendet"

#: camac/instance/milestones/serializers.py
msgid "Statement - preliminary decision"
msgstr "Stellungnahme - Vorentscheid"

#: camac/instance/milestones/serializers.py
msgid "Building decision"
msgstr "Bau- und Einspracheentscheid"

#: camac/instance/milestones/serializers.py
msgid "notice to geometer"
msgstr "Meldung Baubewilligung an Geometer"

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of decision documents"
msgstr "Empfangsbestätigung Entscheiddokumente"

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of preliminary decision"
msgstr "Empfangsbestätigung Vorentscheid"

#: camac/instance/milestones/serializers.py
msgid "notice to gebaeudeschaetzung"
msgstr "Meldung Baubewilligung an Gebäudeschätzung"

#: camac/instance/milestones/serializers.py
msgid "Building permit valid until"
msgstr "Baubewilligung gültig bis"
