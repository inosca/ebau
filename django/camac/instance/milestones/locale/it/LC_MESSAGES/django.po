# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-09-11 17:25+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: camac/instance/milestones/serializers.py
msgid "regular process"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "instance submitted"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Paper submission"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Additional demand"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Check completed"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Instance complete"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Review building commission"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Publication date"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Receipt of objections"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Forwarding to KOOR"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Start circulation"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Distribution completed"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Statement - preliminary decision"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Building decision"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "notice to geometer"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of decision documents"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of preliminary decision"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "notice to gebaeudeschaetzung"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Building permit valid until"
msgstr ""
