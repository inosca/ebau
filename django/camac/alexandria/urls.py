from alexandria.core import views as alexandria_views
from django.urls import path
from rest_framework.routers import SimpleRouter

from camac.alexandria import views

r = SimpleRouter(trailing_slash=False)

# override the default viewset with our patched versions
# this needs to be updated when new urls are added in alexandria
r.register(r"categories", views.PatchedCategoryViewSet)
r.register(r"documents", views.PatchedDocumentViewSet)
r.register(r"files", views.PatchedFileViewSet)
r.register(r"tags", views.PatchedTagViewSet)
r.register(r"tagsynonymgroups", views.PatchedTagSynonymGroupViewSet)
r.register(r"marks", views.PatchedMarkViewSet)

urlpatterns = r.urls

urlpatterns.append(path("hook", alexandria_views.hook_view, name="hook"))
