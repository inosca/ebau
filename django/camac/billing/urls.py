from django.urls import re_path
from rest_framework.routers import SimpleRouter

from camac.billing import views
from camac.billing.export.views import BillingV2EntryExportView

urlpatterns = [
    re_path(
        r"billing-v2-entries/export$",
        BillingV2EntryExportView.as_view(),
        name="billing-export",
    ),
]

r = SimpleRouter(trailing_slash=False)

r.register(r"billing-v2-entries", views.BillingV2EntryViewset, "billing-v2-entry")

urlpatterns.extend(r.urls)
