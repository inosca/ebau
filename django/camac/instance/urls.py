from django.conf import settings
from django.urls import re_path
from rest_framework.routers import SimpleRouter

from . import views


class UnswaggeredInstanceView(views.InstanceView):
    swagger_schema = None


r = SimpleRouter(trailing_slash=False)
urlpatterns = []

r.register(r"instance-states", views.InstanceStateView, "instance-state")
r.register(r"forms", views.FormView, "form")
r.register(r"instances", views.InstanceView, "instance")
r.register(r"form-fields", views.FormFieldView, "form-field")
r.register(
    r"instance-responsibilities",
    views.InstanceResponsibilityView,
    "instance-responsibility",
)
r.register(r"journal-entries", views.JournalEntryView, "journal-entry")
r.register(r"history-entries", views.HistoryEntryView, "history-entry")
r.register(r"issues", views.IssueView)
r.register(r"issue-templates", views.IssueTemplateView, "issue-template")
r.register(r"issue-template-sets", views.IssueTemplateSetView, "issue-template-set")

urlpatterns.append(
    re_path(
        r"form-config",
        views.FormConfigDownloadView.as_view(),
        name="form-config-download",
    )
)


def enable_public_caluma_instances():
    urlpatterns.append(
        re_path(
            r"public-caluma-instances",
            views.PublicCalumaInstanceView.as_view(),
            name="public-caluma-instance",
        )
    )


if settings.APPLICATION.get("ENABLE_PUBLIC_ENDPOINTS"):  # pragma: no cover
    enable_public_caluma_instances()

urlpatterns.extend(r.urls)
