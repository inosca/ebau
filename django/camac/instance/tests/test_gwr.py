import pytest
from django.urls import reverse
from pytest_lazyfixture import lazy_fixture
from rest_framework import status

from camac.instance.tests.test_master_data import (
    gr_master_data_case,  # noqa
    so_master_data_case,  # noqa
    sz_master_data_case_gwr,  # noqa
    sz_master_data_case_gwr_v2,  # noqa
    ur_master_data_case_gwr,  # noqa
)


@pytest.mark.parametrize(
    "canton_name,master_data_case",
    [
        (
            "kt_uri",
            lazy_fixture("ur_master_data_case_gwr"),
        ),
        (
            "kt_gr",
            lazy_fixture("gr_master_data_case"),
        ),
        (
            "kt_schwyz",
            lazy_fixture("sz_master_data_case_gwr"),
        ),
        (
            "kt_schwyz",
            lazy_fixture("sz_master_data_case_gwr_v2"),
        ),
        (
            "kt_so",
            lazy_fixture("so_master_data_case"),
        ),
    ],
)
@pytest.mark.freeze_time("2021-10-07")
def test_gwr_data(
    admin_client,
    canton_name,
    master_data_case,
    settings,
    snapshot,
):
    settings.APPLICATION_NAME = canton_name
    settings.APPLICATION["SHORT_NAME"] = settings.APPLICATIONS[canton_name][
        "SHORT_NAME"
    ]

    url = reverse("instance-gwr-data", args=[master_data_case.instance.pk])

    response = admin_client.get(url)
    assert response.status_code == status.HTTP_200_OK

    snapshot.assert_match(response.json())
