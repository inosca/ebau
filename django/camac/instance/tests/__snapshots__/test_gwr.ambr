# serializer version: 1
# name: test_gwr_data[kt_gr-gr_master_data_case]
  dict({
    'buildingPermitIssueDate': None,
    'client': dict({
      'address': dict({
        'country': dict({
          'countryNameShort': None,
        }),
        'houseNumber': 321,
        'street': 'Testweg',
        'swissZipCode': 4321,
        'town': 'Testingen',
      }),
      'identification': dict({
        'isOrganisation': True,
        'organisationIdentification': dict({
          'organisationName': 'Test AG',
        }),
        'personIdentification': None,
      }),
    }),
    'constructionLocalisation': dict({
      'municipalityName': 'Chur',
    }),
    'constructionProjectDescription': 'Einfamilienhaus',
    'officialConstructionProjectFileNo': '2023-1',
    'projectAnnouncementDate': '2021-03-31',
    'projectCompletionDate': None,
    'projectStartDate': None,
    'realestateIdentification': dict({
      'EGRID': 'CH334687150542',
      'number': 123465,
    }),
    'totalCostsOfProject': 4000,
    'typeOfClient': None,
    'typeOfConstruction': None,
    'typeOfConstructionProject': None,
    'typeOfPermit': None,
    'work': list([
      dict({
        'building': dict({
          'buildingCategory': None,
          'buildingEntrance': None,
          'civilDefenseShelter': None,
          'dateOfConstruction': None,
          'dwellings': list([
          ]),
          'nameOfBuilding': None,
          'numberOfFloors': None,
          'numberOfSeparateHabitableRooms': None,
          'realestateIdentification': None,
          'thermotechnicalDeviceForHeating1': dict({
            'energySourceHeating': 7540,
            'heatGeneratorHeating': 7430,
          }),
          'thermotechnicalDeviceForHeating2': dict({
            'energySourceHeating': 7530,
            'heatGeneratorHeating': 7499,
          }),
          'thermotechnicalDeviceForWarmWater1': dict({
            'energySourceHeating': 7510,
            'heatGeneratorHotWater': 7651,
          }),
          'thermotechnicalDeviceForWarmWater2': dict({
            'energySourceHeating': 7512,
            'heatGeneratorHotWater': 7660,
          }),
        }),
        'kindOfWork': None,
      }),
    ]),
  })
# ---
# name: test_gwr_data[kt_schwyz-sz_master_data_case_gwr]
  dict({
    'buildingPermitIssueDate': None,
    'client': dict({
      'address': dict({
        'country': dict({
          'countryNameShort': 'ch',
        }),
        'houseNumber': None,
        'street': 'Teststrasse 3',
        'swissZipCode': 5678,
        'town': 'Musterdorf',
      }),
      'identification': dict({
        'isOrganisation': False,
        'organisationIdentification': dict({
          'organisationName': None,
        }),
        'personIdentification': dict({
          'firstName': 'Max',
          'officialName': 'Mustermann',
        }),
      }),
    }),
    'constructionLocalisation': None,
    'constructionProjectDescription': 'Grosses Haus',
    'officialConstructionProjectFileNo': None,
    'projectAnnouncementDate': None,
    'projectCompletionDate': None,
    'projectStartDate': None,
    'realestateIdentification': dict({
      'EGRID': 'CH1234567890',
      'number': 1234,
    }),
    'totalCostsOfProject': 129000,
    'typeOfClient': 6161,
    'typeOfConstruction': None,
    'typeOfConstructionProject': None,
    'typeOfPermit': None,
    'work': list([
      dict({
        'building': dict({
          'buildingCategory': 1060,
          'buildingEntrance': None,
          'civilDefenseShelter': None,
          'dateOfConstruction': None,
          'dwellings': list([
            dict({
              'floor': None,
              'floorNumber': None,
              'floorType': None,
              'kitchen': True,
              'locationOfDwellingOnFloor': 'Nord',
              'multipleFloor': True,
              'noOfHabitableRooms': 4,
              'surfaceAreaOfDwelling': 42,
              'usageLimitation': None,
            }),
          ]),
          'nameOfBuilding': None,
          'numberOfFloors': 2,
          'numberOfSeparateHabitableRooms': None,
          'realestateIdentification': None,
          'thermotechnicalDeviceForHeating1': dict({
            'energySourceHeating': 7540,
            'heatGeneratorHeating': 7436,
          }),
          'thermotechnicalDeviceForHeating2': None,
          'thermotechnicalDeviceForWarmWater1': dict({
            'energySourceHeating': 7560,
            'heatGeneratorHotWater': None,
          }),
          'thermotechnicalDeviceForWarmWater2': None,
        }),
        'kindOfWork': None,
      }),
    ]),
  })
# ---
# name: test_gwr_data[kt_schwyz-sz_master_data_case_gwr_v2]
  dict({
    'buildingPermitIssueDate': None,
    'client': dict({
      'address': dict({
        'country': dict({
          'countryNameShort': 'ch',
        }),
        'houseNumber': None,
        'street': 'Teststrasse 3',
        'swissZipCode': 5678,
        'town': 'Musterdorf',
      }),
      'identification': dict({
        'isOrganisation': False,
        'organisationIdentification': dict({
          'organisationName': None,
        }),
        'personIdentification': dict({
          'firstName': 'Max',
          'officialName': 'Mustermann',
        }),
      }),
    }),
    'constructionLocalisation': None,
    'constructionProjectDescription': 'Grosses Haus',
    'officialConstructionProjectFileNo': None,
    'projectAnnouncementDate': None,
    'projectCompletionDate': None,
    'projectStartDate': None,
    'realestateIdentification': dict({
      'EGRID': 'CH1234567890',
      'number': 1234,
    }),
    'totalCostsOfProject': 129000,
    'typeOfClient': 6161,
    'typeOfConstruction': None,
    'typeOfConstructionProject': None,
    'typeOfPermit': None,
    'work': list([
      dict({
        'building': dict({
          'buildingCategory': 1020,
          'buildingEntrance': None,
          'civilDefenseShelter': True,
          'dateOfConstruction': None,
          'dwellings': list([
            dict({
              'floor': None,
              'floorNumber': None,
              'floorType': None,
              'kitchen': True,
              'locationOfDwellingOnFloor': 'West',
              'multipleFloor': False,
              'noOfHabitableRooms': 2,
              'surfaceAreaOfDwelling': 70,
              'usageLimitation': None,
            }),
            dict({
              'floor': None,
              'floorNumber': None,
              'floorType': None,
              'kitchen': False,
              'locationOfDwellingOnFloor': 'Ost',
              'multipleFloor': False,
              'noOfHabitableRooms': 3,
              'surfaceAreaOfDwelling': 24,
              'usageLimitation': None,
            }),
          ]),
          'nameOfBuilding': 'Grosses Haus',
          'numberOfFloors': 4,
          'numberOfSeparateHabitableRooms': 24,
          'realestateIdentification': None,
          'thermotechnicalDeviceForHeating1': dict({
            'energySourceHeating': 7510,
            'heatGeneratorHeating': 7411,
          }),
          'thermotechnicalDeviceForHeating2': None,
          'thermotechnicalDeviceForWarmWater1': dict({
            'energySourceHeating': 7570,
            'heatGeneratorHotWater': 7650,
          }),
          'thermotechnicalDeviceForWarmWater2': None,
        }),
        'kindOfWork': None,
      }),
    ]),
  })
# ---
# name: test_gwr_data[kt_so-so_master_data_case]
  dict({
    'buildingPermitIssueDate': None,
    'client': dict({
      'address': dict({
        'country': dict({
          'countryNameShort': 'ch',
        }),
        'houseNumber': '565',
        'street': 'Anthony Unions',
        'swissZipCode': 7429,
        'town': 'Schultzbury',
      }),
      'identification': dict({
        'isOrganisation': True,
        'organisationIdentification': dict({
          'organisationName': 'Wells, Arroyo and Sims',
        }),
        'personIdentification': None,
      }),
    }),
    'constructionLocalisation': dict({
      'municipalityName': 'Solothurn',
    }),
    'constructionProjectDescription': 'Grosses Haus',
    'officialConstructionProjectFileNo': '2024-1',
    'projectAnnouncementDate': '2024-02-22',
    'projectCompletionDate': None,
    'projectStartDate': None,
    'realestateIdentification': dict({
      'EGRID': 'CH123456789',
      'number': 123456789,
    }),
    'totalCostsOfProject': 129000,
    'typeOfClient': None,
    'typeOfConstruction': 6235,
    'typeOfConstructionProject': 6011,
    'typeOfPermit': None,
    'work': list([
      dict({
        'building': dict({
          'buildingCategory': 1060,
          'buildingEntrance': dict({
            'locality': dict({
              'swissZipCode': '4500',
            }),
          }),
          'civilDefenseShelter': None,
          'dateOfConstruction': None,
          'dwellings': list([
            dict({
              'floor': 3102,
              'floorNumber': 2,
              'floorType': 3101,
              'kitchen': True,
              'locationOfDwellingOnFloor': 'SÃ¼d',
              'multipleFloor': True,
              'noOfHabitableRooms': '20',
              'surfaceAreaOfDwelling': '420',
              'usageLimitation': 3401,
            }),
            dict({
              'floor': 3100,
              'floorNumber': None,
              'floorType': 3100,
              'kitchen': False,
              'locationOfDwellingOnFloor': 'Nord',
              'multipleFloor': False,
              'noOfHabitableRooms': '10',
              'surfaceAreaOfDwelling': '72',
              'usageLimitation': 3402,
            }),
          ]),
          'nameOfBuilding': 'Villa',
          'numberOfFloors': None,
          'numberOfSeparateHabitableRooms': None,
          'realestateIdentification': dict({
            'EGRID': 'CH123456789',
            'number': 123456789,
          }),
          'thermotechnicalDeviceForHeating1': dict({
            'energySourceHeating': 7511,
            'informationSourceHeating': 869,
            'revisionDate': None,
          }),
          'thermotechnicalDeviceForHeating2': dict({
            'energySourceHeating': 7570,
            'informationSourceHeating': 869,
            'revisionDate': None,
          }),
          'thermotechnicalDeviceForWarmWater1': dict({
            'energySourceHeating': 7511,
            'informationSourceHeating': 869,
            'revisionDate': None,
          }),
          'thermotechnicalDeviceForWarmWater2': dict({
            'energySourceHeating': 7580,
            'informationSourceHeating': 869,
            'revisionDate': None,
          }),
        }),
        'kindOfWork': 6001,
      }),
    ]),
  })
# ---
# name: test_gwr_data[kt_uri-ur_master_data_case_gwr]
  dict({
    'buildingPermitIssueDate': '2021-07-20',
    'client': dict({
      'address': dict({
        'country': dict({
          'countryNameShort': 'ch',
        }),
        'houseNumber': 123,
        'street': 'Teststrasse',
        'swissZipCode': 1233,
        'town': 'Musterdorf',
      }),
      'identification': dict({
        'isOrganisation': True,
        'organisationIdentification': dict({
          'organisationName': 'ACME AG',
        }),
        'personIdentification': None,
      }),
    }),
    'constructionLocalisation': dict({
      'municipalityName': 'Altdorf',
    }),
    'constructionProjectDescription': 'Grosses Haus',
    'officialConstructionProjectFileNo': '1201-21-003',
    'projectAnnouncementDate': '2021-07-16',
    'projectCompletionDate': '2021-08-05',
    'projectStartDate': '2021-07-25',
    'realestateIdentification': dict({
      'EGRID': 'CH123456789',
      'number': 123456789,
    }),
    'totalCostsOfProject': 129000,
    'typeOfClient': 6141,
    'typeOfConstruction': 6235,
    'typeOfConstructionProject': 6011,
    'typeOfPermit': 5031,
    'work': list([
      dict({
        'building': dict({
          'buildingCategory': 1060,
          'buildingEntrance': None,
          'civilDefenseShelter': None,
          'dateOfConstruction': dict({
            'yearMonthDay': '2021-08-05',
          }),
          'dwellings': list([
            dict({
              'floor': 3102,
              'floorNumber': 2,
              'floorType': 3101,
              'kitchen': True,
              'locationOfDwellingOnFloor': 'SÃ¼d',
              'multipleFloor': True,
              'noOfHabitableRooms': '20',
              'surfaceAreaOfDwelling': '420',
              'usageLimitation': 3401,
            }),
            dict({
              'floor': 3100,
              'floorNumber': None,
              'floorType': 3100,
              'kitchen': False,
              'locationOfDwellingOnFloor': 'Nord',
              'multipleFloor': False,
              'noOfHabitableRooms': '10',
              'surfaceAreaOfDwelling': '72',
              'usageLimitation': 3402,
            }),
          ]),
          'nameOfBuilding': 'Villa',
          'numberOfFloors': None,
          'numberOfSeparateHabitableRooms': None,
          'realestateIdentification': dict({
            'EGRID': 'CH123456789',
            'number': 123456789,
          }),
          'thermotechnicalDeviceForHeating1': dict({
            'energySourceHeating': 7570,
            'informationSourceHeating': 869,
            'revisionDate': '2021-10-07',
          }),
          'thermotechnicalDeviceForHeating2': None,
          'thermotechnicalDeviceForWarmWater1': None,
          'thermotechnicalDeviceForWarmWater2': dict({
            'energySourceHeating': 7520,
            'informationSourceHeating': 869,
            'revisionDate': '2021-10-07',
          }),
        }),
        'kindOfWork': 6001,
      }),
    ]),
  })
# ---
